src/w3c/seo.js

```diff
@@ -33,6 +33,18 @@ export const requiresCanonicalLink = new Set([
 ]);
 
 export async function run(conf) {
+  if (conf.gitRevision) {
```

Comment:
I wonder if it should be a part of `conf.github` though - but that can mean we lose setting `conf.github` as string. Maybe better as `gitRevision`.